src/main/java/edu/greenblitz/robotName/commands/shooter/MoveShooterToAngle.java

```diff
@@ -0,0 +1,20 @@
+package edu.greenblitz.robotName.commands.shooter;
+
+import edu.greenblitz.robotName.commands.shooter.pivot.MovePivotIfInDangerZone;
+import edu.greenblitz.robotName.commands.shooter.pivot.MovePivotToAngle;
+import edu.greenblitz.robotName.subsystems.arm.Elbow;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.*;
+
+public class MoveShooterToAngle {
+
+	public static Command getCommand(Rotation2d targetAngle) {
```

Comment:
get a shooter position and make an overload for a rotation 2d

Reply:
no because you would want to move the shooter to any angle and not just preset positions

Reply:
i made another getCommand